src/System.Memory/src/System/SpanHelpers.cs

```diff
@@ -84,6 +96,8 @@ public static class PerTypeValues<T>
             //
             public static readonly bool IsReferenceOrContainsReferences = IsReferenceOrContainsReferencesCore(typeof(T));
 
+            public static readonly bool IsReference = IsReferenceCore(typeof(T));
```

Comment:
Nit - why not call it `IsValueType` to make same as the reflection API?

`public static readonly bool IsValueType = type.GetTypeInfo().IsValueType;`